PyFlyt/gym_envs/simple_waypoint.py

```diff
@@ -30,7 +30,7 @@ class SimpleWaypointEnv(PyFlytEnv):
     def __init__(
         self,
         num_targets: int = 4,
-        use_yaw_targets: bool = True,
+        use_yaw_targets: bool = False,
```

Comment:
Don't modify this file.

Reply:
Not using this file anymore, will revert to original.